config/nginx.conf

```diff
@@ -1,6 +1,12 @@
 # This is the version used in development environments
 server {
     listen 8079;
+    server_name micromasters.herokuapp.com;
+    return 301 https://micromasters.mit.edu$request_uri;
+}
+
+server {
+    listen 8079 default_server;
```

Comment:
Correct on both points. I had added it in for testing locally. It doesn't hurt to leave it in, but it's not required for dev environments.